workshop/content/docs/api-deep-dive/features.md

```diff
@@ -174,17 +174,17 @@ the request
 A client application can then retrieve the GeoJSON document and display
 or process it.
 
-### Client Usage
+### Client usage
 
 Various clients/softwares supports OGC API- Features, You can checkout latest list [here](https://github.com/opengeospatial/ogcapi-features/blob/master/implementations/clients/README.md)
```

Comment:
@krishnaglodha  We already have a list of implementations, [further up](https://github.com/opengeospatial/ogcapi-workshop/commit/1b45e2e36dcbdbaf77643d49addd953a2d4f93f5#diff-8fdc87e09dc02c51fa15700cf6d04ac3245d4c33377f84a405fd89d0bef54f67R63) on this page :) No need to repeat it, here!

Reply:
Good catch, I think we wanted a specific "client" perspective. +1 to remove pygeoapi from the Client usage sections.